web/modules/custom/ct_manager/src/Plugin/QueueWorker/ProcessUsers.php

```diff
@@ -90,6 +90,9 @@ public function processItem($data) {
     /** @var \Drupal\ct_manager\ContributionSourceInterface $plugin_instance */
     $plugin_instance = $this->pluginManager->createInstance($plugin_id);
     if (!($plugin_instance->isUserValid($user))) {
+      if ($plugin_id === "github") {
+        return;
+      }
```

Comment:
This is a code smell. The `ct_manager` should not have to know about `ct_github`.